mentorApp/lib/FTU/Filters/DropDownPage.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:mentorApp/FTU/Filters/DropDownButton.dart';
+
+class DropDownPage extends StatelessWidget {
+  //static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      //title: _title,
+      home: Scaffold(
+        //appBar: AppBar(title: const Text(_title)),
```

Comment:
It's actually beneficial to include the `appBar` in our screens because not all phones have uniform screen shapes (e.g. [iPhones](https://miro.medium.com/max/3200/0*uf2Y4BXGSd_1yQoI) etc). Having a buffer between for the top is a good and is widely used in apps.

I did in the `Dashboard.dart` file like this:
```dart
appBar: AppBar(
     backgroundColor: Colors.transparent,
     elevation: 0.0,
)
```